openandroidinstaller/assets/configs/lancelot.yaml

```diff
@@ -0,0 +1,53 @@
+metadata:
+  maintainer: A non (anon)
+  brand: xiaomi
+  device_name: Xiaomi Redmi 9 / Poco M2
+  is_ab_device: false
+  device_code: lancelot
+  supported_device_codes:
+    - lancelot
+    - galahad
+    - shiva
+  untested: true
+steps:
+  unlock_bootloader:
+    - type: confirm_button
+      content: >
+        As a first step, you need to unlock the bootloader. A bootloader is the piece of software, that tells your phone
+        how to start and run an operating system (like Android). Your device should be turned on. This will reset your phone.
+    - type: link_button_with_confirm
+      content: >
+        - Create a Mi account on Xiaomi’s website. Beware that one account is only allowed to unlock one unique device every 30 days.
+
+        - Add a phone number to your Mi account, insert a SIM into your phone.
+
+        - Enable developer options in `Settings` > `About Phone` by repeatedly tapping MIUI Version.
+
+        - Link the device to your Mi account in `Settings` > `Additional settings` > `Developer options` > `Mi Unlock status`.
+
+        - Download the Mi Unlock app with the link bellow (Windows is required to run the app), and follow the instructions provided by the app. It may tell you that you have to wait, usually 7 days. If it does so, please wait the quoted amount of time before continuing to the next step!
+
+        - After device and Mi account are successfully verified, the bootloader should be unlocked.
+
+        - Since the device resets completely, you will need to re-enable USB debugging to continue : `Settings` > `Additional settings` > `Developer options` > `USB debugging`
+      link: https://en.miui.com/unlock/download_en.html
+  boot_recovery:
+    - type: confirm_button
+      content: >
+        Now you need to boot a custom recovery system on the phone. A recovery is a small subsystem on your phone, that manages updating,
+        adapting and repairing of the operating system.
+    - type: call_button
+      content: >
+        Once the device is fully booted, you need to reboot into the bootloader again by pressing 'Confirm and run' here. Then continue.
+      command: adb_reboot_bootloader
```

Comment:
In the LOS instructions it somehow uses adb -d reboot bootloader. Might not be important, I just noticed it here for the first time. It seems to be used to direct the command to the device, which should be the default.

Reply:
Yep, seen that too. It should work without, but we may add "-d" in OAI for every "adb_reboot_bootloader", and maybe others adb commands too ?

Reply:
hm, it seems to be on all new instructions, so maybe there has been an update to adb :) I will investigate. Should be fine here.